src/Components/PropertyInspector/PropertyInspector.stories.local.tsx

```diff
@@ -43,7 +43,7 @@ export const AdtTwin = (args, { globals: { theme, locale } }) => {
 AdtTwin.argTypes = {
     twinId: {
         control: { type: 'text' },
-        defaultValue: 'PasteurizationMachine_A01'
+        defaultValue: 'ActivePowerSensorTwin'
```

Comment:
wouldn't you want to revert the value back. The ActivePowerSensor was just put there just so you can repro the error locally, correct?

Reply:
done!